go/enclave/components/rollup_consumer.go

```diff
@@ -65,9 +64,9 @@ func NewRollupConsumer(
 
 func (rc *rollupConsumerImpl) ProcessL1Block(b *common.BlockAndReceipts) (*common.ExtRollup, error) {
 	stopwatch := measure.NewStopwatch()
-	defer rc.logger.Info("Block processed", log.BlockHashKey, b.Block.Hash(), log.DurationKey, stopwatch)
+	defer rc.logger.Info("Rollup consumer processed block", log.BlockHashKey, b.Block.Hash(), log.DurationKey, stopwatch)
 
-	rollups := rc.extractRollups(b, rc.storage)
+	rollups := rc.extractRollups(b)
```

Comment:
there is no such thing as a `canonical rollup` any more. The `getCanonicaRollup` function should be removed, and all rollups found in a block should be processed normally.

Reply:
Lets ignore this as I've planned to touch those in the next PR. This one just changes to use sequence numbers instead of the hashes and lays the foundation